packages/core-typings/src/ISession.ts

```diff
@@ -30,7 +30,10 @@ export interface ISession {
 	host: string;
 	ip: string;
 	loginAt: Date;
+	logoutAt?: Date;
 	closedAt?: Date;
+	logoutBy?: string;
```

Comment:
```suggestion
	logoutBy?: IUser['_id'];
```
When the attribute name doesn't specify what kind of information is in the attribute, it's a good practice to link the type directly like this so we don't have to look up if this uses an id or an username or something else.